lib/buffer.js

```diff
@@ -1040,9 +1048,9 @@ Buffer.prototype.toJSON = function toJSON() {
 };
 
 function adjustOffset(offset, length) {
-  // Use Math.trunc() to convert offset to an integer value that can be larger
+  // Use Math.MathTrunc() to convert offset to an integer value that can be larger
```

Comment:
This might be an unnecessary change.
```suggestion
  // Use Math.trunc() to convert offset to an integer value that can be larger
```

Reply:
Not nit, and thank you😂

Reply:
By the way, can you help me trigger the Jenkins CI?